config/environments/development.rb

```diff
@@ -17,6 +17,7 @@
   config.action_mailer.raise_delivery_errors = false
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = { address: "localhost", port: 1025 }
+  config.action_mailer.default_url_options = {:host => "localhost:3000"}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.